### DISCUSSION TITLE
Update the binary on NixOS

### DISCUSSION THREAD

**Discussion by Eloquencere** [JanDeDobbeleer/oh-my-posh#5780] [General]
I use the nix package manager on Ubuntu 24.04 and observed that the latest version of Oh-My-Posh is 19.29.0. However, the official GitHub releases binary has moved on to 23.20.3, which is also reflected on the AUR. Therefore, I would like to ask the developers to please update it on Nix to benefit the rest of us. GitHub releases cannot be updated automatically via a package manager so, the rest of us will miss out on crucial updates going forward

**Comment by JanDeDobbeleer** [comment]
@Eloquencere I don't maintain that package unfortunately.
